keyboards/buildakb/potato65hs/readme.md

```diff
@@ -0,0 +1,19 @@
+# Potato65 Hotswap Keyboard
+
+![Potato65 Hotswap](https://i.imgur.com/ysAGZCmh.jpg)
+
+A hot-swappable gasket mount stacked acrylic 65% keyboard
+
+-   Keyboard Maintainer: [Maelkk](https://github.com/Aeonstrife)
+-   Hardware Supported: Potato65 Keyboard
+-   Hardware Availability: Private Group-Buy
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to jump to the bootloader?

Reply:
Added instructions on how to switch to bootloader mode.